### PR TITLE
Remove gendered pronouns

### PR DESCRIPTION
**Pull request by boennemann**
While reviewing the latest additions to the npm section I noticed the use of unnecessarily gendered pronouns like "his". In an effort to make these guides accessible and welcoming for non-male people this PR removes this language.
Using the the handy tool [alex](https://www.npmjs.com/package/alex) I tried to find all occurrences.

It's output before:

```
README.md: no issues found
WERE_HIRING.md: no issues found
analysis/profiling.md: no issues found
collaboration/bug-reports.md: no issues found

collaboration/contributing.md
  127:72-127:75  warning  `his` may be insensitive, use `their`, `theirs`, `them` instead  her-him

collaboration/issues.md
  19:123-19:128  warning  `crazy` may be insensitive, use `rude`, `mean`, `disgusting`, `vile`, `person with symptoms of mental illness`, `person with mental illness`, `person with symptoms of a mental disorder`, `person with a mental disorder` instead  nuts
  21:113-21:117  warning  `he's` may be insensitive, use `they`, `it` instead                                                                                                                                                                                 he-she
  21:132-21:135  warning  `his` may be insensitive, use `their`, `theirs`, `them` instead                                                                                                                                                                     her-him

collaboration/versioning.md: no issues found
git/general.md: no issues found
npm/access.md: no issues found

npm/publishing.md
  41:69-41:72  warning  `his` may be insensitive, use `their`, `theirs`, `them` instead  her-him

style-guides/coffeescript.md: no issues found
style-guides/commonJS.md: no issues found
style-guides/markdown.md: no issues found
style-guides/scss.md: no issues found

⚠ 5 warnings
```

### CODE REVIEW

File: collaboration/issues.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Issues can have more than one label. The labels we use are:
 
 Workflow labels allow quickly scanning the status of tickets in a list and filtering issues. For example, create a filter for all the things that are waiting on your review.
 
-- **Discussion**: Label for discussions about features, workflows, technical challenges, research, etc. Bring in all your crazy ideas!
+- **Discussion**: Label for discussions about features, workflows, technical challenges, research, etc. Bring in all your ideas!
 - **Needs planning**: When a task is unclear and needs planning before you can start. Assign it to the person who is responsible for the specifications.
-- **For review**: Add this label when you requested a review from someone. The reviewer removes this label when he's finished with his comments.
+- **For review**: Add this label when you requested a review from someone. The reviewer removes this label when they're finished with their comments.
```
Comment (marcbachmann) [inline collaboration/issues.md]: ~~The reviewer removes this label when they're finished with their comments.~~
The **reviewers** remove this label when they're finished with their comments.

**Comment by mthpvg** [comment]
This is good, thanks for that.

File: collaboration/issues.md (hunk 1 above)
Comment (boennemann) [inline collaboration/issues.md]: The sentence before is talking about a single reviewer, which is why the following sentence is singular as well. They is used as a [singular, neutral pronoun](https://en.wikipedia.org/wiki/Singular_they) in this case.

**Comment by boennemann** [comment]
With a +1 from @mthpvg I'm merging.

File: collaboration/issues.md (hunk 1 above)
Comment (marcbachmann) [inline collaboration/issues.md]: :+1: Thanks for the reference

**Pull request merged**